wscl-issues/draft/sequence-definition

```diff
@@ -0,0 +1,69 @@
+Issue:          SEQUENCE-DEFINITION
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         draft
+Edit History:   16-May-24, Version 1 by Jin-Cheng Guu
+References:     SEQUENCE
+
+Problem Description:
+
+  In the draft ANSI Common Lisp specification, the description of the system
+  class SEQUENCE [a] conflicts with the description of the Sequence Concepts
+  [b], where
+
+    [a]: The types vector and the type list are disjoint subtypes of type
+    sequence, but are not necessarily an exhaustive partition of sequence.
+
+    [b]: A sequence is an ordered collection of elements, implemented as
+    either a vector or a list.
+
+Proposal:
+
+  This proposal changes the description of the Sequence Concepts so that [b]
+  instead reads:
+
+    > A sequence is an ordered collection of elements, implemented either as a
+    vector, a list, or possibly something else.
```

Comment:
We could probably say "some other implementation-defined object" to be a bit more formal. Maybe type instead of object?